File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/entities/cs/createupdateentitymetadata4.cs
// <snippetcreateupdateentitymetadata4>


 CreateAttributeRequest createCheckedDateRequest = new CreateAttributeRequest
 {
  EntityName = _customEntityName,
  Attribute = new DateTimeAttributeMetadata
  {
   SchemaName = "new_checkeddate",
   RequiredLevel = new AttributeRequiredLevelManagedProperty(AttributeRequiredLevel.None),
   Format = DateTimeFormat.DateOnly,
   DisplayName = new Label("Date", 1033),
   Description = new Label("The date the account balance was last confirmed", 1033)

  }
 };

 _serviceProxy.Execute(createCheckedDateRequest);
 Console.WriteLine("An date attribute has been added to the bank account entity.");

// </snippetcreateupdateentitymetadata4>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/entitiesvb/vb/createupdateentitymetadata4.vb
' <snippetcreateupdateentitymetadata4>


 Dim createCheckedDateRequest As CreateAttributeRequest = New CreateAttributeRequest With {
  .EntityName = _customEntityName,
  .Attribute = New DateTimeAttributeMetadata With {
   .SchemaName = "new_checkeddate",
   .RequiredLevel = New AttributeRequiredLevelManagedProperty(AttributeRequiredLevel.None),
   .Format = DateTimeFormat.DateOnly,
   .DisplayName = New Label("Date", 1033),
   .Description = New Label("The date the account balance was last confirmed", 1033)
  }
 }

 _serviceProxy.Execute(createCheckedDateRequest)
 Console.WriteLine("An date attribute has been added to the bank account entity.")

' </snippetcreateupdateentitymetadata4>